data/loot_lists/level10/level10_armor.yml

```diff
@@ -13,6 +13,7 @@ weighted_entries:
       exceptional: 5
       fine: 5
     adjective2:
+      unidentified: 12

```

Comment:
is this mutually exclusive with the other adjective2 entries?  Shouldn't it be its own adjective3 entry?

Reply:
We won't ever allow multiple bonuses? I added it here and implemented the re-roll in the assumption that we might want to allow multiple bonuses in the future.